Readme.md

```diff
@@ -48,19 +48,43 @@ var obj = {};
 pointer.set(obj, '/example/bla', 'hello');
 ```
 
-
 ### .remove(object, pointer)
 
-Removes an attribute of object referenced by pointer.
+Removes an attribute of object or an item from array referenced by pointer.
+**Note**: Since `v0.6.0` this function changed handles arrays differently, see # .
+In order to have previous behaviour you can simply replace `remove` with `delete`.
 
 ```Javascript
 var obj = {
     example: 'hello'
 };
 pointer.remove(obj, '/example');
 // obj -> {}
+
+var array = ['foo', 'bar'];
+pointer.remove(array, '/0');
+// array -> ['bar']
 ```
 
+### .delete(object, pointer)
+
+Use JS `delete` operator on an attribute of object or an item from array referenced by pointer.
```

Comment:
@epoberezkin I don't think it will be useful to anybody. If you new to this library it just confuses you. If used it prior `0.6.0` and want to know why started to behave differently `remove` you will check description of `remove`.
\+ `delete` isn't a hack to simplify migration it's has it own semantic. And useful on it's own without any ties to `0.6.0`.